react-native-libraries.json

```diff
@@ -10016,5 +10016,12 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/KroosX4V/react-native-orientation-manager",
+    "npmPkg": "react-native-orientation-manager",
```

Comment:
```suggestion
```

This field can be skipped, since the repository name matches package name, it's a field dedicated mostly for scoped packages.